example/src/Screens/SimpleStack.tsx

```diff
@@ -66,12 +66,13 @@ const NewsFeedScreen = ({
   route,
   navigation,
 }: StackScreenProps<SimpleStackParams, 'NewsFeed'>) => {
+  let x: 'Albums' | 'Article';
   return (
     <ScrollView>
       <View style={styles.buttons}>
         <Button
           mode="contained"
-          onPress={() => navigation.navigate('Albums')}
+          onPress={() => navigation.navigate(x)}
```

Comment:
`x` is never assigned, so this will cause a runtime error

Reply:
oops that was debug code